code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1005,7 +1005,7 @@
 	// Allows you to put in item-specific reactions based on species
 	if(user != H)
 		user.do_attack_animation(H)
-	if(H.check_shields(I.force, "the [I.name]", I, 0, I.armour_penetration))
+	if(user != H && H.check_shields(I.force, "the [I.name]", I, 0, I.armour_penetration))
```

Comment:
Is this the only thing that makes it so that you can't trigger it yourself any more? Personally I'd rather have this in a separate PR after this one gets merged. Keep the balance and refactor separate, you know?